### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by subsymbolic**
Reviewer: Chris
Asana: https://app.asana.com/0/414709148257752/446023679873951

**Description**:
• Updates test script so content blocking integration tests also run on travis
• Adds additional urls to content blocking tests to ensure all lists are exercised
• Fixes broken integration tests

**Steps to test this PR**:
TEST 1. Run the run_tests.sh script to ensure this works correctly
TEST 2. Check that integration tests ran successfully on travis (this branch ran them)

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR DRI Checklist:
- [x] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications (Database connections, Grafana stats, CPU)

### CODE REVIEW

File: IntegrationTests/ContentBlockerTests.swift [hunk 1]
```diff
@@ -119,9 +120,13 @@ class ContentBlockerTests: XCTestCase {
...
     }
+ 
+    private func waitForLaunch() {
+        sleep(10)
+    }
     
-    private func waitForPageTitle() {
-        sleep(2)
+    private func waitForPageLoad() {
+        sleep(6)
```
Comment (brindy) [inline IntegrationTests/ContentBlockerTests.swift]: Optional improvement: Have a look in SnapshotHelper.swift for a way to watch the loading indicator.  You'll probably have to add your own timeout though.

Comment (subsymbolic) [reply IntegrationTests/ContentBlockerTests.swift]: I like that, thanks!

**Comment by subsymbolic** [comment]
@brindy the tests are passing but I've removed them from travis for now. Travis is too brittle.

**Comment by brindy** [comment]
All good by me

**Pull request merged**